libioc/Jail.py

```diff
@@ -1603,6 +1609,25 @@ def devfs_ruleset(self) -> libioc.DevfsRules.DevfsRuleset:
         if self._allow_mount_zfs == "1":
             devfs_ruleset.append("add path zfs unhide")
 
+        if self.config["jail_zfs"] is True:
```

Comment:
do i understand the PR correctly that this can be both, a zfs dataset or a zvol?